src/viewer_window.cpp

```diff
@@ -372,6 +386,13 @@ class DatarefViewerWindow {
 		last_regex = 0 != XPGetWidgetProperty(regex_toggle_button, xpProperty_ButtonState, nullptr);
 		last_changed = 0 != XPGetWidgetProperty(change_filter_button, xpProperty_ButtonState, nullptr);
 
+        FILE * pFile;
+        pFile = fopen ("./Resources/plugins/DataRefTool/drtpref.txt","w+");
```

Comment:
I like to keep the prefs file in the plugin folder so it does not get lost in the ./Output/preferences folder. It is a method I have used for a long time over all platforms.